src/pages/Task.tsx

```diff
@@ -71,13 +90,23 @@ export const Task: React.FC = () => {
     status === TaskStatus.Succeeded ||
     status === TaskStatus.SetupFailed ||
     status === TaskStatus.SystemFailed ||
-    status === TaskStatus.TestTimedOut
+    status === TaskStatus.TestTimedOut ||
+    error
   ) {
     stopPolling();
   }
 
+  if (error) {
+    return (
+      <PageWrapper>
+        <Banners banners={bannersState} removeBanner={dispatchBanner.remove} />
```

Comment:
What if we integrated the banner into the PageWrapper directly? It would save allot of boiler plate

Reply:
this is a great idea

Reply:
i will make ticket for this

Reply:
https://jira.mongodb.org/browse/EVG-7978